lang/en/docs/accounts/service-levels.md

```diff
@@ -1,61 +1,61 @@
-| Reserved Compute Nodes |	The number of supercomputing clusters which are made available to the users of the account | 
-| Month Data Usage | The total quota of storage space which is allocated to the account on a monthly basis |
-| Private Data  | The possibility to hold private entities inside an account. Otherwise entities are accessible for other platform users to view. More explanation [here](../collaboration/sharing/access-levels.md). |
-| Organizations/Teams  | The possibility to create organization accounts, and multiple teams inside each of these organizations |
-| Maximum Team Size  | The maximum number of users that can be hosted inside teams in general |
-| Support Level	| The different forms and depths of support that users will receive by our support staff in case of technical difficulties being encountered |
-| Command Line Access  |  The possibility to access the platform interface via command line |
 
-<sup class="c-red">1</sup> Ordinary = general case without the usage of any saving, premium or specialized type hardware. 
 
-## Types
+[^1]: The total quota of storage space which is allocated to the account on a monthly basis.
 
-For the [personal accounts](overview.md#personal-accounts) we offer "Promo", "Advanced", and "Pro" service levels. For the [organizational accounts](overview.md#personal-accounts) we offer "Promo", "Enterprise" and "Enterprise-Extra" levels. The "Promo" service level is designed as a temporary promotional way for the users to understand the operations of our platform. For the detailed comparison of the pricing and features associated with the different Service Levels offered as part of our platform, the user is referred to the [pricing documentation page](../pricing/service-levels.md). 
+[^2]: The possibility to hold private entities inside an account. Otherwise entities are accessible for other platform users to view. More explanation [here](../collaboration/sharing/access-levels.md).
 
-## View/Change Service Level
+[^3]: The maximum number of users that can be hosted inside an account.
 
-All the relevant information concerning Service Levels, under the context of the currently selected Account, can be inspected through the corresponding [component of the User Interface](ui/service-level.md). It is important to bear in mind that, when dealing with collaborative accounts, only the Owner or an Administrator of the account is given the rights to undergo such operations on behalf of the whole Organization. 
+[^4]: The different forms and depths of support that users will receive by our support staff in case of technical difficulties being encountered.
+Support requests with Normal, High, and Urgent support severity are handled withing 48, 24, and 12 business hours respectively.
 
-## How Service Levels are set
+[^5]: The maximum number of entities such as projects, materials, and workflows that can be hosted inside an account.
```

Comment:
We should also explain that these number can be lifted on a case-by-case basis when a sufficient explanation is provided

Reply:
Added.